sources/Cargo.lock

```diff
@@ -118,14 +134,14 @@ dependencies = [
 
 [[package]]
 name = "actix-server"
-version = "1.0.3"
+version = "1.0.4"
```

Comment:
I'm worried about the seemingly unrelated updates to dependencies here.  It means we're compiling two versions of *their* dependencies, and gives me discomfort about compatibility and testing.  I'd prefer this be done as part of a separate dep update PR, which I was going to work on soon.  Can we minimize the Cargo.lock change here?  (I usually try updating Cargo.toml as necessary, then doing a `cargo check` to update Cargo.lock... there may be other tricks to not updating dependencies...)

Same with `tools/`.

Reply:
I agree, the build was very difficult though and everything I did was in service of getting it to build. Once this is reference crates.io instead of git for tough and coldsnap, I will make some effort to undoing as much of the lock churn as I can. I was getting 'failed to satisfy dependency' issues for a few things.

One possibility might be to bump deps separately, rebase, and take it from there as I guess the issues stemmed from newer requirements in tough and/or coldsnap than bottlerocket (?)

Reply:
(deps have been bumped separately)